### ISSUE TITLE
Re-review for rest-client-java guide

### ISSUE THREAD

**Issue by Kubik42** [OpenLiberty/guide-rest-client-java#13]
PR: #14 

**What you'll learn**
- ~"You will learn how to access a REST service and deserialize the returned JSON resources containing a list of artists and their albums using two different approaches." -> too wordy~ **(n/a: previously addressed)**.
- ~"The REST service that provides the artists and albums resources has already been written for you and is accessible at:" -> only accessible once its running.~ **(n/a: previously addressed)**
- ~"Which responds with the following JSON:" -> "Which will respond ..."~ **(n/a: previously addressed)**

**Starting the service**
- ~merge this section into "Getting started" and make it a requirement instead of "if you want to" -> the users must start the service~ **(n/a - does not follow current template for guides)**.
- ~"When the server is running, you can find your service at:" -> removed the bullet from the link~ **(n/a: previously addressed)**

**POJOs**
- ~"start **by** creating..."~ **(n/a: previously addressed)**
- ~"If the getter and setter signatures..." -> "If the getter and setter **method** signatures..."~
- ~"Note that the Artist POJO, does not require this annotation since all instance members match the JSON keys by name." -> needs a bit of rewording + maybe move below the bullet~ **(n/a: previously addressed)**

**Consuming the REST resource**
- ~"This method makes a GET request to the running artist service and retrieves the JSON." -> as a client~ **(n/a: previously addressed)**
- ~add "Create" to "start/src/main/java/io/openliberty/guides/consumingrest/service/ArtistResource.java" and the rest~ **(n/a: previously addressed)**
- ~"which has been done for you:" -> remove this or reword the sentence to put more emphasis at the dependency already being included~ **(n/a: previously addressed)**
- ~"To use Jackson’s JAX-RS JSON provider, you must explicitly register it like the following line since automatic discovery and registration are not available in the JAX-RS client API:" -> reword~ **(n/a: previously addressed)**

**Testing**
- [x] "Maven finds and executes all tests under it/ and each test method must be marked with the @Test annotation." -> remove (removed as unnecessary) 
- ~add path to class above the code snippet~ **(n/a: previously addressed)**
- ~split stopping the server from running everything~ **(n/a: previously addressed)**
- ~replace "mvn clean install" with "mvn verify"?~ **(n/a: previously addressed)**
- ~"If the tests pass, you **will** see the following output:"~ **(n/a: previously addressed)**

**Comment by gkwan-ibm** [comment]
close
- the guide was updated to use dev mode
